middleware/limiter/README.md

```diff
@@ -36,9 +36,7 @@ app.Use(limiter.New(limiter.Config{
 	},
 	Max:          20,
 	Duration:     30 * time.Second,
-	Key:          func(c *fiber.Ctx) string {
-		return c.Get("x-forwarded-for")
-	},
+	KeyGenerator: func(c *fiber.Ctx) string, 
```

Comment:
We'll need a valid function, this is just a signature. Maybe

```go
KeyGenerator: func(c *fiber.Ctx) string {
  return "key"
},
```

Reply:
i will keep the default, return c.IP(), what you think ?